WcaOnRails/app/controllers/api/internal/v1/payment_controller.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+class Api::Internal::V1::PaymentController < Api::Internal::V1::ApiController
+  def init
+    return json: { error: "Missing fields" } unless params["attendee_id"].present? && params["amount"].present?
+
+    holder = AttendeePaymentRequest.new(attendee_id: params["attendee_id"])
```

Comment:
`Model.new(...)` does create a Ruby object instance but does not actually save it to the database